client/src/components/Navbar/Navbar.js

```diff
@@ -60,8 +63,8 @@ function Navbar() {
   const [authenticated, setAuthenticated] = useState(true);
   const [profileMenuAnchor, setProfileMenuAnchor] = useState(null);
   // TODO: get profileID and avatar from user
-  // const profileID = 1234;
-  const profileName = "Test User";
+  //   const profileID = store.getState().id;
+  const profileName = store.getState().name;
```

Comment:
This doesn't get automatically updated if the Redux state changes. You probably want to use https://react-redux.js.org/api/hooks. This would also remove the need for a separate `authenticated` state.

Reply:
Fixed